include/identity_key.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef IDENTITY_KEY_H_
+#define IDENTITY_KEY_H_
+
+
+/**
+ * @file
+ * @defgroup identity_key Identity key APIs
+ * @{
+ *
+ * @brief API for identity key on CryptoCell devices with KMU
+ */
+
+
+#include "hw_unique_key.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Identity key size in bytes, corresponding to ECC secp256r1 */
+#define IDENTITY_KEY_SIZE_BYTES                 (32)
+
+/** @brief Error value when MKEK is not written in KMU */
+#define ERR_IDENTITY_KEY_MKEK_MISSING           (0x15501)
+
+/** @brief Error value when identity key is missing in KMU */
+#define ERR_IDENTITY_KEY_MISSING                (0x15501)
```

Comment:
Is this correct? same as line 30

Reply:
This is a copy paste issue, I changed the values to be unique in all the cases.